TerrorConsole/Assets/_Main/Scripts/Audio/AudioMusicStart.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class AudioMusicStart : MonoBehaviour
+    {
+        private void Start()
+        {
+            AudioManager.Source.PlayMusicPiano();
+        }
+    }
```

Comment:
Use `enum` and `switch` to make a generic script or change the name of the script to know it triggers that specific music.